src/sentry/integrations/jira/notify_action.py

```diff
@@ -16,39 +16,43 @@
 from sentry.utils.http import absolute_uri
 from sentry.web.decorators import transaction_start
 
+from .utils import get_name_for_jira
+
 logger = logging.getLogger("sentry.rules")
 
+INTEGRATION_KEY = "integration"
+
 
 class JiraNotifyServiceForm(forms.Form):
-    jira_integration = forms.ChoiceField(choices=(), widget=forms.Select())
+    integration = forms.ChoiceField(choices=(), widget=forms.Select())
```

Comment:
I think this would require a migration but we can just wipe out any rules that cause problems.

Reply:
Is anyone actually using these except us? We can just delete and recreate the rules.

Reply:
Yeah it's just us.